### PR TITLE
Allow for trailing backslashes in project path

### PR DESCRIPTION
**Pull request by kflores772**

### CODE REVIEW

File: routes/create.js [hunk 1]
```diff
@@ -22,18 +22,22 @@ module.exports = {
 
     get: function (req, res) {
         var cmd = "create",
-            args = req.query.args,
+            location = path.resolve(req.query.location),
+            projectId = "\"" + req.query.projectId + "\"",
```
Comment (rcruz) [inline routes/create.js:27]: You wont need these extra quotes here

File: routes/create.js [hunk 2]
```diff
@@ -22,18 +22,22 @@ module.exports = {
...
+            name = "\"" + req.query.name + "\"",
             cmdPath = path.resolve(__dirname, path.join("..", "..", "webworks")),
             execStr,
             child;
 
-            execStr = util.format('"%s" %s %s', cmdPath, cmd, args);
+            execStr = util.format('"%s" %s "%s"', cmdPath, cmd, location, projectId, name);
```
Comment (rcruz) [inline routes/create.js]: If the extra quotes are placed here instead

**Comment by kflores772** [comment]
@rcruz projectId and name are optional. Therefore I did not add two more %s placeholders that are quoted.

**Comment by rcruz** [comment]
@kflores772 I see. Makes sense to me. Can we put a comment in the code in case we forget the reasoning in the future?

**Comment by kflores772** [comment]
Comment has been added.

**Comment by kflores772** [comment]
@rcruz support for mac/linux added

**Comment by rcruz** [comment]
r+, tested and working on mac

**Pull request merged**